### ISSUE TITLE
Remove or update the example NGINX confs

### ISSUE THREAD

**Issue by aentwist** [nextcloud/docker#1954]
The example nginx confs are quite outdated (w.r.t. .htaccess on nextcloud/server) - I suggest we remove the nginx confs from here since they are a maintenance problem and instead link to the example nginx conf(s) in the nextcloud documentation and say, go get it from there. the examples here led me very astray while I was porting it to Caddy.

Although there is no guarantee that the docs are up to date (they probably aren't 100%) it is more likely they will be as that is where most people go, and it leaves one less place to maintain it.

**Comment by aentwist** [comment]
Related to https://github.com/nextcloud/docker/issues/928

**Comment by aentwist** [comment]
In the spirit of ain't nobody got time for this, I heavily suggest removing them unless good counter-motivation, for keeping them, is provided

**Comment by joshtrichards** [comment]
I agree with the general idea and the goal, but the issue is that those aren't just standalone example config files for nginx - they're dependencies for the docker-compose build example in the parent folder(s)[1]. So any consideration to removing the nginx stuff will require a solution to that.

[1] e.g. 
- https://github.com/nextcloud/docker/blob/289f0bb8a3f1bd24d1633bbd01798c5476368827/.examples/docker-compose/with-nginx-proxy/postgres/fpm/docker-compose.yml
- https://github.com/nextcloud/docker/blob/289f0bb8a3f1bd24d1633bbd01798c5476368827/.examples/docker-compose/with-nginx-proxy/postgres/fpm/web/Dockerfile

**Comment by aentwist** [comment]
I made this suggestion with full awareness of that 🙂 It is certainly the main consideration. It might not be worth much, but in my opinion the additional steps users would need to take in order to run examples are outweighed by the maintenance benefits.

> and say, go get it from there

This does require more of users. Would involve adding documentation to the examples on the additional step to get them running. The main problem with this is it runs the risk of the linked NGINX conf not working. Which would be surprising, but is certainly an issue.

Honestly I'd really like to `wget` from a permalink but of course that isn't trivial since the confs from the docs are wrapped into that page? Would be really nice if there was a GitHub project for them since they are out-of-scope of nextcloud/server, where we could get raw files... e.g. nextcloud/configs?

Needs more thought... hm
